python/triton/runtime/build.py

```diff
@@ -6,23 +6,26 @@
 from .. import knobs
 
 
-def _build(name, src, srcdir, library_dirs, include_dirs, libraries):
+def _build(name: str, src: str, srcdir: str, library_dirs: list[str], include_dirs: list[str],
```

Comment:
If we decide to land https://github.com/triton-lang/triton/pull/6704 then we can add this file to the list of typechecked files.